scania-digital-design-system/sdds
# feat:(checkbox) migrate checkbox component

**Describe pull-request**  
Migration of checkbox component. 

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: [AB#2386](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2386)

**How to test**  
_Add description how to test if possible_
1. Go to the storybook link below
2. Check in components -> checkbox
3. Check that the design aligns with the one in figma.

**Screenshots**  
-

**Additional context**  
-
